packages/client/src/forms/register/mappings/query/registration-mappings.ts

```diff
@@ -157,24 +162,38 @@ export const userTransformer =
     if (!_.history) {
       return
     }
-    const history: History = [..._.history]
+    const history = [...(_.history as History[])]
```

Comment:
`_` typically signifies an unused variable. Maybe we could rename it for instance `event`?